packages/operator/pkg/webserver/routes/v1/training/model_training_test.go

```diff
@@ -94,6 +94,31 @@ func (s *ModelTrainingRouteSuite) SetupSuite() {
 		// If we get a panic that we have a test configuration problem
 		panic(err)
 	}
+
+	// Create the connection that will be used as the outputConnection param for a training.
+	if err := s.connRepository.CreateConnection(&connection.Connection{
+		ID: testMtOutConn,
+		Spec: odahuflowv1alpha1.ConnectionSpec{
+			Type: connection.GcsType,
+		},
+	}); err != nil {
+		// If we get a panic that we have a test configuration problem
+		panic(err)
+	}
+
+	// Create the connection that will be used as the default outputConnection param for a training.
+	if err := s.connRepository.CreateConnection(&connection.Connection{
+		ID: testMtOutConnDefault,
+		Spec: odahuflowv1alpha1.ConnectionSpec{
+			Type: connection.GcsType,
+		},
+	}); err != nil {
+		// If we get a panic that we have a test configuration problem
+		panic(err)
+	}
+
+	// Define output connection in configuration to not require this from user requests
+	viper.Set(train_config.OutputConnectionName, testMtOutConnDefault)
```

Comment:
Reset the packaging_config.OutputConnectionName viper value to the nil in the test tear down.